lib/ouija_tests.ml

```diff
@@ -1,74 +1,96 @@
-open OUnit2
+open Core.Std
+open Async.Std
+open Sentinel.Std
 
-let rest _ =
+let check exp op = fun _ -> return (exp = op ())
```

Comment:
Do you need to use async for these tests? Since the previous version was using Ounit I kind of expect Sentinel.Basic to be enough